### PR TITLE
Removes embed link from paragraph

### LINKED ISSUE #210: Remove embedded video from paragraph body
Until we resolve #185 we want to remove the embedded video option from the paragraph body.

### PR DESCRIPTION
**Pull request by sheralim012**
Fixes #210 

- Removed embed feature from paragraph of article in wagtail CMS

### CODE REVIEW

**Comment by sheralim012** [comment]
@dannyparsons kindly check and let me know.

File: home/migrations/0037_auto_20210701_1317.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            model_name='article',
+            name='body',
+            field=wagtail.core.fields.StreamField([('heading', wagtail.core.blocks.CharBlock(form_classname='full title')), ('paragraph', wagtail.core.blocks.RichTextBlock(features=['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'bold', 'italic', 'ol', 'ul', 'hr', 'link', 'document-link', 'image'])), ('markdown', wagtailmarkdown.blocks.MarkdownBlock(icon='code')), ('image', wagtail.images.blocks.ImageChooserBlock()), ('list', wagtail.core.blocks.ListBlock(wagtail.core.blocks.CharBlock(label='Item'))), ('numbered_list', wagtail.core.blocks.ListBlock(wagtail.core.blocks.CharBlock(label='Item'))), ('page_button', wagtail.core.blocks.StructBlock([('page', wagtail.core.blocks.PageChooserBlock()), ('text', wagtail.core.blocks.CharBlock(max_length=255, required=False))])), ('embedded_poll', home.blocks.EmbeddedQuestionnaireChooserBlock(page_type=['questionnaires.Poll'])), ('embedded_survey', home.blocks.EmbeddedQuestionnaireChooserBlock(page_type=['questionnaires.Survey'])), ('embedded_quiz', home.blocks.EmbeddedQuestionnaireChooserBlock(page_type=['questionnaires.Quiz'])), ('media', home.blocks.MediaBlock(icon='media'))]),
```
Comment (dannyparsons) [inline home/migrations/0037_auto_20210701_1317.py:21]: We only have h2, h3, and h4 currently in paragraph, so let's keep it as that.

Comment (sheralim012) [reply home/migrations/0037_auto_20210701_1317.py:21]: ok

Comment (sheralim012) [reply home/migrations/0037_auto_20210701_1317.py]: done

**Pull request merged**